xcp_d/interfaces/bids.py

```diff
@@ -165,28 +166,40 @@ def _run_interface(self, runtime):
                 f"{hstr}.sphere.reg2",
             )
 
-            # TODO: Collect from templateflow once it's uploaded.
-            # MCRIBS: tpl-fsaverage_hemi-?_den-41k_desc-reg_sphere.surf.gii
-            self._results["source_sphere"] = os.path.join(
-                self.inputs.segmentation_dir,
-                "templates_fsLR",
-                f"tpl-fsaverage_hemi-{hemisphere}_den-41k_desc-reg_sphere.surf.gii",
+            # Load the fsaverage-41k sphere.
+            self._results["source_sphere"] = str(
+                get_template(
+                    template="fsaverage",
+                    space=None,
+                    hemi=hemisphere,
+                    density="41k",
+                    desc="reg",
+                    suffix="sphere",
+                )
             )
```

Comment:
There is no `tpl-fsaverage_hemi-<L|R>_den-41k_desc-reg_sphere.surf.gii` on TemplateFlow. Do we need to add this, or is it perhaps just the same as `tpl-fsaverage_hemi-<L|R>_den-41k_sphere.surf.gii`?

Reply:
Also, should this really be fsaverage and not dhcpSym?